strings/test.js

```diff
@@ -31,13 +31,14 @@ describe('Strings - basics', () => {
     });
   });
 
-  xdescribe('join', () => {
+  describe('join', () => {
     it('should join the 2 strings together', () => {
       expect(join('hello ' ,'there')).toEqual('hello there');
+      expect(join(1,2)).toEqual(3); // Could potentially test for arrays too?
```

Comment:
Ahhh I've just spent a bit of time writing my comment above and you've written a passing test for this case! Think about this for a second - is this correct? We're expecting to "join" 2 strings together, you are adding them together 🤔 

Refer to my earlier comment about this then come back and have another look

We don't want to worry about arrays for now, we can come back to them later